### ISSUE TITLE
Add a min height?

### ISSUE THREAD

**Issue by elfeffe** [Gustavinho/laravel-views#103]
<img width="1717" alt="68c8132afcb63203761426f664e878cc" src="https://user-images.githubusercontent.com/4717507/119632749-64374f80-be11-11eb-8ffa-e42a20d5e38d.png">

I think you should add a min-h-screen to the components

**Comment by Gustavinho** [comment]
Hi @elfeffe I saw your PR, I'll merge it

**Comment by df999** [comment]
Hey guys - I can understand why you want a min height, but now this causes other issues. It creates a huge gap below the views. Eg. try creating two views one after another when there are minimal rows and there will be a huge gap between them.

Maybe there's another way to fix filter dropdown? Or at least have an easy way to remove the min height?

**Comment by elfeffe** [comment]
Paste some screenshot please. I will check it.

**Comment by df999** [comment]
![table views](https://user-images.githubusercontent.com/3520592/120861048-b2a4c680-c554-11eb-9ee8-9f55c0777024.jpg)
The height of the gap is relative to the screen height of course.

In the blade file, I just have two components, one after another:
```
<livewire:forms.project-table-view />
<livewire:forms.project-table-view />
```

**Comment by Gustavinho** [comment]
Thanks @df999 for raising this, yeah the `min-height` was to avoid the filter cutting off but you are right, there should be a better way to fix this, sorry for this issue I should have tested it this way, I will add this test scenario before releasing a new version

**Comment by df999** [comment]
Okay thanks. Perhaps there's a z-index solution to have it on top of the content below.

**Comment by Gustavinho** [comment]
Hi @elfeffe what version of the package and browser did you use to see this issue? by any chance are you using your own tailwindcss file?

**Comment by elfeffe** [comment]
@Gustavinho Are you sure the question is for me?
I'm using Latest Chrome and Firefox, with default TailwindCSS, local, not from CDN.
The package, I was using the latest stable version.

**Comment by Gustavinho** [comment]
Yeah @elfeffe, thanks, I'm trying to fix the original issue but I haven't been able to replicate it again and I just wanted to be sure that I'm using the correct environment to replicate it

I've tested it with
- LaravelViews 2.2.4, 2.2.3, 2.2.2, 2.2.1, 2.2.0
- Local Tailwindcss build with/without purging LaravelViews files, 
- LaravelViews default tailwindcss file
- Chrome, Firefox

This is how it looks
![image](https://user-images.githubusercontent.com/4228757/121120259-d3824b80-c7e2-11eb-9802-d16b38a27535.png)


And this is how I'm rendering the table view
```html
<body class="bg-gray-100">
        <div class="bg-white">
            <livewire:my-users-table-view />
        </div>
</body>
```

Not sure if I'm missing something else to see this behavior, do you have something different?

**Comment by var2611** [comment]
@df999 is right about space but it is something else I think.

I've started using this after this update so I don't know what has changed what.

![Screenshot (340)](https://user-images.githubusercontent.com/29687965/147966481-15d688a8-7353-4375-b0ed-6e7600c27cc8.png)

Above image is toolbar.blade.php from component/toolbar .

and issue is : <div> used for search bar and filter 
`line 11 : <div class="flex-1">`
`line 16 : <div class="flex space-x-1 flex-1 justify-end items-center mb-4">`

![image](https://user-images.githubusercontent.com/29687965/147967098-702a83ca-ba3a-4be3-ad1a-0e253b90351d.png)

![Screenshot (342)](https://user-images.githubusercontent.com/29687965/147967145-34c08cc7-9fcc-4c7b-b505-5db00ab572e8.png)

Look above SS this two line are printing when there is no search and filter enabled.

I haven't forked any project yet otherwise I'll submit request. Maybe another time. Thanks

**Comment by Gustavinho** [comment]
just in case it is useful for anyone, this happened to me last week it turns out I had a div wrapping my component and the div has a `overflow-hidden` class, not saying this is what happened to others, but this can be one of the reasons
